src/main/java/com/actiontech/dble/plan/common/item/function/mathsfunc/ItemFuncConv.java

```diff
@@ -25,6 +26,9 @@ public final String funcName() {
 
     @Override
     public String valStr() {
+        if (args.size() == 1) {
```

Comment:
does it need to be handled separately BIN()?

Reply:
doc   :    https://dev.mysql.com/doc/refman/5.7/en/string-functions.html#function_bin

content ： Returns a string representation of the binary value of N, where N is a longlong (BIGINT) number. This is equivalent to CONV(N,10,2). Returns NULL if N is NULL.